Resources/Prototypes/_DeltaV/Objectives/traitor.yml

```diff
@@ -0,0 +1,31 @@
+﻿# Kill
+- type: entity
+  parent: [BaseTraitorObjective, BaseTeachLessonObjective]
+  id: TeachLessonRandomPersonObjective
+  description: Kill them, and show everyone we mean business. They only need to die once.
+  components:
+  - type: Objective
+    difficulty: 1.75
+    unique: false
+  - type: TargetObjective
+    title: objective-condition-teach-person-title
+  - type: PickRandomPerson
+  - type: TeachLessonCondition
```

Comment:
In https://github.com/DeltaV-Station/Delta-v/blob/master/Resources/Prototypes/DeltaV/Objectives/traitor.yml there is no `TeachLessonCondition` component in the `TeachLessonRandomPersonObjective` entity. Which makes sense, as it's defined in one of the parents -- `BaseTeachLessonObjective`.

Is it needed here?